cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -29,6 +29,7 @@
 #include <modmesh/buffer/pymod/buffer_pymod.hpp> // Must be the first include.
 
 #include <modmesh/buffer/pymod/array_common.hpp>
+
```

Comment:
Do not change formatting in unrelated files.  Adding/removing blank lines is refactoring.  Refactoring change should not be included in the same PR as feature development.